torchci/clickhouse_queries/queue_time_analysis/queue_time_per_job_name/params.json

```diff
@@ -0,0 +1,11 @@
+{
+    "params": {
+        "startTime": "DateTime64",
+        "endTime": "DateTime64",
+        "granularity": "String",
+        "items": "Array<String>",
+        "repos": "Array<String>",
+        "workflowNames": "Array<String>"
```

Comment:
I think these should technically be Array(String) but idk if it actually has an effect